modules/va_forms/app/serializers/va_forms/form_list_serializer.rb

```diff
@@ -13,9 +13,5 @@ class FormListSerializer < ActiveModel::Serializer
     def id
       object.row_id
     end
-
-    def last_sha256_change
-      object.versions.last.created_at&.strftime('%Y-%m-%d') if object&.versions&.last&.created_at
-    end
```

Comment:
This method isn't needed now that the value is coming from the `last_sha256_change` column on the record. I've confirmed that it's still returned in the form detail response. (Notice it is included in the `attributes` section of this serializer.)